app.config.ts

```diff
@@ -1,108 +1,109 @@
+    }),
+    slug: appEnv.select({
+      development: 'react-native-starter-dev',
+      staging: 'react-native-starter-stg',
+      production: 'react-native-starter-prod'
+    }),
+    scheme: appEnv.select({
+      development: 'rnstarterdev',
+      staging: 'rnstarterstg',
+      production: 'rnstarter'
+    }),
+    owner: 'ronas_it',
+    version: '0.0.1',
+    runtimeVersion: {
+      policy: 'sdkVersion'
+    },
+    updates: {
+      url: `https://u.expo.dev/${projectId}`
+    },
+    orientation: 'portrait',
     backgroundColor: '#000000',
-    config: {
-      usesNonExemptEncryption: false
-    }
-  },
-  android: {
-    versionCode: 1,
-    package: 'com.ronasit.rnstarter.dev',
-    permissions: []
-  },
-  web: {
-    favicon: './src/assets/images/favicon.png'
-  },
-  packagerOpts: {
-    config: 'metro.config.js',
-    sourceExts: ['ts', 'tsx', 'js', 'jsx', 'json', 'wasm', 'svg']
-  },
-  // TODO: Configure this to use Sentry or remove
-  // hooks: {
-  //   postPublish: [
-  //     {
-  //       file: 'sentry-expo/upload-sourcemaps',
-  //       config: {
-  //         url: 'https://your-sentry-url',
-  //         organization: 'your-sentry-organization',
-  //         project: 'your-sentry-project',
-  //         authToken: 'your-sentry-token'
-  //       }
-  //     }
-  //   ]
-  // },
-  // plugins: ['sentry-expo'],
-  extra: defaultAppEnvConfig
+    icon: './src/assets/images/icon.png',
+    splash: {
+      image: './src/assets/images/splash.png',
+      resizeMode: 'contain',
+      backgroundColor: '#000000'
+    },
+    assetBundlePatterns: [
+      './src/assets/images/*',
+      './src/libs/account-access/assets/images/*',
+      './src/assets/fonts/*',
+      './src/assets/i18n/*'
+    ],
+    userInterfaceStyle: 'dark',
+    ios: {
+      buildNumber: '1',
+      supportsTablet: false,
+      bundleIdentifier: appEnv.select({
+        development: 'com.ronasit.rnstarter.dev',
+        staging: 'com.ronasit.rnstarter.stg',
+        production: 'com.ronasit.rnstarter'
+      }),
+      backgroundColor: '#000000',
+      config: {
+        usesNonExemptEncryption: false
+      }
+    },
+    android: {
+      versionCode: 1,
+      package: appEnv.select({
+        development: 'com.ronasit.rnstarter.dev',
+        staging: 'com.ronasit.rnstarter.stg',
+        production: 'com.ronasit.rnstarter'
+      }),
+      permissions: []
+    },
+    web: {
+      favicon: './src/assets/images/favicon.png'
+    },
+    packagerOpts: {
+      config: 'metro.config.js',
+      sourceExts: ['ts', 'tsx', 'js', 'jsx', 'json', 'wasm', 'svg']
+    },
+    // TODO: Configure this to use Sentry or remove
+    // hooks: {
+    //   postPublish: [
+    //     {
+    //       file: 'sentry-expo/upload-sourcemaps',
+    //       config: {
+    //         url: 'https://your-sentry-url',
+    //         organization: 'your-sentry-organization',
+    //         project: 'your-sentry-project',
+    //         authToken: 'your-sentry-token'
```

Comment:
Could you please remove this? Now we store auth token in [env variable](https://docs.expo.dev/guides/using-sentry/#:~:text=token%20through%20the-,SENTRY_AUTH_TOKEN,-environment%20variable%20in) within project secrets